java/ql/lib/semmle/code/java/JDK.qll

```diff
@@ -4,6 +4,7 @@
 
 import Member
 import semmle.code.java.security.ExternalProcess
+private import semmle.code.java.dataflow.FlowSteps
```

Comment:
I presume this is a pretty large import, I hope it's fine and won't break anything. Should I do this differently?

Reply:
This is ok -- note that in FlowSteps.qll's `Frameworks` module you should private-import this file back. This is to ensure all queries using FlowSteps see the same set of standard value-preserving methods etc, and so the related QL can be evaluated once for the whole query suite, not re-evaluated per query as it would need to be if each one defined extra flow steps.

Reply:
Done. Thanks!